src/parser.rs

```diff
@@ -201,7 +194,8 @@ fn parse_complex(input: &[u8]) -> IResult<&[u8], Instruction> {
                 },
             ))
         }
-        _ => unreachable!(),
+        // Invalid op code
+        250..=255 => Err(nom::Err::Failure((input, nom::error::ErrorKind::TooLarge))),
```

Comment:
The error returned here is a little hacky, but giving a real error (something like `DviInvalidOpCode`) would involve making a big change to the library: we would need to define a custom error type for the library and use that in `IResult`.

Reply:
Your chose of error is correct IMO - I don't think it's worth creating a new error type.